pex/tools/commands/graph.py

```diff
@@ -68,7 +68,12 @@ def _create_dependency_graph(pex):
                 graph.add_edge(
                     start=dist.project_name,
                     end=req.project_name,
-                    label=str(req) if (req.specifier or req.marker) else None,
+                    label="{specifier}{marker}".format(
+                        specifier=req.specifier if req.specifier else "",
+                        marker="; {}".format(req.marker) if req.marker else "",
+                    )
+                    if (req.specifier or req.marker)
+                    else None,
```

Comment:
Formatting nit that I'm more saying as an FYI: if you wrap this whole thing in parentheses, Black will format this block with proper indentation.

Reply:
I think we've had this chat before - I consider whatever an auto-formatter does as proper - that's the very deal you make when you sign up for auto formatting - you let go of formatting.

Reply:
Yeah, you're right, we have. I view the extra parentheses as giving a small hint to Black so that it can do its thing. But that's fine - not blocking of course.